packages/cx-editors/doc/RichText.md

```diff
@@ -18,38 +18,40 @@ Content Editor Details](../RichText#content-editor-details).
 
 ### Usage of the CX Rich Text Editor
 
-The following can be applied to any Slot as in the follow example.
+What's shown in the following example can be applied to any Slot.
 
-    ```js
-      Editors: {
-        Content: as(cxRichText.Default)(RichTextClean);
-      },
-      Schema: {
-        Content: withNode('content'),
-      }
-    ```
+```js
+  Editors: {
+    Content: as(cxRichText.Default)(RichTextClean);
+  },
+  Schema: {
+    Content: withNode('content'),
+  }
+```
 
-We recommend the Editor component & tokens are defined in Editors Domain and what node is defined in Schema Domain.
+We recommend that the Editor Component and tokens be defined in the Editors Domain, and the node be
+defined in the Schema Domain.
 
 ### Overriding Rich Text Editor
 
-#### For Overriding specific existing styles usined CanvasX Rich Text Editor
+#### Overriding Specific Existing Styles Using CanvasX Rich Text Editor
 
-See [CanvasX Site Typography](../CX_Elements/CX_SiteTypography)
+See [CanvasX Site Typography](../../CX_Elements/CX_SiteTypography).
 
-#### Shadowing *Preferred method
+#### Shadowing (*Preferred Method)
```

Comment:
Move Shadowing first as its preferred method.
Also its not clear these are two different options. 
Maybe prefix each with "via "